google/cloud/storage/benchmarks/benchmark_utils.h

```diff
@@ -79,6 +82,27 @@ std::string BuildUsage(std::vector<OptionDescriptor> const& desc,
 std::vector<std::string> OptionsParse(std::vector<OptionDescriptor> const& desc,
                                       std::vector<std::string> argv);
 
+class SimpleTimer {
+ public:
+  SimpleTimer() = default;
+
+  void Start();
+  void Stop();
+
+  std::chrono::microseconds elapsed_time() const { return elapsed_time_; }
```

Comment:
Perhaps a note that these are only valid after `Stop()`?

Reply:
Done.